templates/registration/login-scripts.html

```diff
@@ -0,0 +1,35 @@
+{% load static %}
+<!-- begin::Global Config(global config for global JS sciprts) -->
+<script>
+    var KTAppOptions = {
```

Comment:
Do we need those scripts ?

Reply:
If we don't use those scripts, `Sign Up` and `Forgot Password` buttons will not work. Should I remove their block from login page?

Reply:
Lets keep those for the time being.